src/types.rs

```diff
@@ -474,8 +473,7 @@ pub struct RuntimeConfig {
 	#[serde(default)]
 	#[serde(with = "port_range_format")]
 	pub http_server_port: (u16, u16),
-	pub ipfs_seed_randomize: bool,
-	pub ipfs_seed: u64,
+	pub ipfs_seed: Option<String>,
```

Comment:
If we decide for the seed not to have 'random' option, we could set this var to be Option<u64>, in which case we wouldn't need the `integer_part` func.